### PR TITLE
Added the Wasserstein loss 

### PR DESCRIPTION
**Pull request by ChernovAndrey**
Hello Team,

I have been experimenting with your model and made a modification by replacing the cross-entropy loss with the Wasserstein loss to better account for the topology of time series data. While I lack the resources to train the model from scratch, I fine-tuned your T5-small model using both the Wasserstein loss and the cross-entropy loss. The results are promising. If you’re interested in the theoretical background, you can refer to this paper: https://arxiv.org/abs/2409.15367.

Additionally, I’ve made a few minor changes, which are documented in the README of my forked repository. I’m unsure if you’d consider these changes valuable for incorporation into your repository, but I wanted to share them in case this approach aligns with your interests.

In any case thank you for your excellent work!

Best regards,
Andrei Chernov

### CODE REVIEW

**Comment by abdulfatir** [comment]
Hi Andrei,

Thank you for sharing this with us. It looks interesting indeed. We are keeping this repository as an archive for research around Chronos models from us. Therefore, I am afraid that it may not be the best idea to merge such major changes into this repository. I would encourage you start a new thread under [discussions](https://github.com/amazon-science/chronos-forecasting/discussions) to share your work with the community.
